src/main/resources/edu/cornell/kfs/coa/document/workflow/CuObjectCodeActivationGlobalMaintenanceDocument.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>OCAG</name>
+      <parent>COA</parent>
+      <label>Object Code Activation Global</label>
+      <authorizer>org.kuali.kfs.krad.workflow.authorizer.CfDocumentTypeAuthorizer</authorizer>
+      <docHandler>${kfs.url}/objectcodeactivationglobalmaintenance.do?methodToCall=docHandler</docHandler>
+      <active>true</active>
+      <routingVersion>1</routingVersion>
```

Comment:
I think you can either remove this element or change the routingVersion to 2.

Reply:
ok